File: services/src/main/java/com/liferay/mobile/sdk/v7/permission/PermissionService.java
package com.liferay.mobile.sdk.v7.permission;

import com.liferay.mobile.sdk.Call;
import com.liferay.mobile.sdk.annotation.Param;
import com.liferay.mobile.sdk.annotation.Path;
import com.liferay.mobile.sdk.http.Response;
import java.lang.String;

@Path("/permission")
public interface PermissionService {
  @Path("/check-permission")
  Call<Response> checkPermission(@Param(name = "groupId") long groupId, @Param(name = "name") String name, @Param(name = "primKey") String primKey);
}

File: windows/Liferay.SDK/Service/V70/Permission/PermissionService.cs
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace Liferay.SDK.Service.V70.Permission
{
	public class PermissionService : ServiceBase
	{
		public PermissionService(ISession session)
			: base(session)
		{
		}

		public async Task CheckPermissionAsync(long groupId, string name, string primKey)
		{
			var _parameters = new JsonObject();

			_parameters.Add("groupId", groupId);
			_parameters.Add("name", name);
			_parameters.Add("primKey", primKey);

			var _command = new JsonObject()
			{
				{ "/permission/check-permission", _parameters }
			};

			await this.Session.InvokeAsync(_command);
		}
	}
}